torchvision/transforms/functional.py

```diff
@@ -760,7 +760,7 @@ def adjust_hue(img: Tensor, hue_factor: float) -> Tensor:
     raise TypeError('img should be PIL Image. Got {}'.format(type(img)))
 
 
-def adjust_gamma(img, gamma, gain=1):
+def adjust_gamma(img, gamma: float, gain: float = 1):
```

Comment:
```suggestion
def adjust_gamma(img: Tensor, gamma: float, gain: float = 1) -> Tensor:
```
Like other methods :)

Reply:
I was a bit confused by that annotation in the other functions in this file, doesn't this take and return _either_ a tensor or PIL image?

Reply:
Yes, the input can be either tensor or PIL image, jit does not recognize `Union`. So, it is sort of partial type annotation...

Reply:
If we want to enforce the type hints, through a linter or a future version of python, won't this fail?

Reply:
So, type hints are necessary for torchscript only for the argument types, but mypy requires return types as well and we had issues in the past where importing the code to fbcode would fail due to missing return types in mypy, see https://github.com/pytorch/vision/pull/1696

For now, given that torchscript doesn't support `Union`, I think this is the best we can do, but once they improve support for additional types we should revisit those annotations